packages/cli/src/commands/deploy.ts

```diff
@@ -42,10 +43,11 @@ export default class Deploy extends Command {
         }
         this.debug(`Loaded CARE package root: ${carePackage.packageRoot}`)
         this.debug(`Elements: ${JSON.stringify(carePackage.elements, null, 2)}`)
+        const only = flags.only ? flags.only.split(",") : null;
```

Comment:
Might be good to have some validation here, for example if the user pass input as --only="Solution1, Solution2" the result only array will be ["Solution1", " Solution2"], i.e the second one will contain spaces. The other option will be to trim the string after split.

Reply:
Good point, added trimming.